packages/wrangler/src/index.tsx

```diff
@@ -264,14 +264,19 @@ export async function main(argv: string[]): Promise<void> {
           default: "worker",
         })
         .positional("template", {
-          describe: "a link to a GitHub template",
+          describe: "The URL of a GitHub template",
           default: "https://github.com/cloudflare/worker-template",
```

Comment:
won't we need a w2 version of this repository as well?

Reply:
Really, for the default, we should just encourage the use of `wrangler init`.

Reply:
If we are hitting the default then we should just encourage the use of `wrangler init`, right?.